aten/src/ATen/native/cpu/UnaryOpsKernel.cpp

```diff
@@ -116,6 +116,13 @@ static void neg_kernel(TensorIterator& iter) {
   });
 }
 
+static void erfinv_kernel(TensorIterator& iter) {
```

Comment:
I had to implement this method to be able to use dispatch. It will implemented when merging with correspondent CPU PR: https://github.com/pytorch/pytorch/pull/24908